app/views/partials/chargeSummary/ChargeSummaryPaymentAllocationTable.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.financialDetails.ChargeItem
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import _root_.implicits.ImplicitCurrencyFormatter.CurrencyFormatter
+@import _root_.implicits.ImplicitDateFormatterImpl
+
+@this(govukTable: GovukTable,
+        implicitDateFormatter: ImplicitDateFormatterImpl)
+
+@(chargeItem: ChargeItem, link: String, linkText: String)(implicit messages: Messages)
+
+@import implicitDateFormatter.longDate
+
+<h2 class="govuk-heading-l" id="allocation"> @messages("chargeSummary.allocation")</h2>
+
+<table class="govuk-table" id="allocation-table">
+    <thead class="govuk-table__head">
+        <tr class="govuk-table__row" id="heading-row">
+            <th scope="col" class="govuk-table__header">@messages("chargeSummary.whereMoneyWent")</th>
+            <th scope="col" class="govuk-table__header">@messages("chargeSummary.date")</th>
+            <th scope="col" class="govuk-table__header govuk-table__header--numeric">@messages("chargeSummary.paymentAmount")</th>
+        </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+        <tr class="govuk-table__row" id="table-row-1">
+            <td class="govuk-table__cell">
+                <a class="govuk-link" id="poa-allocation-link" href="@link">
+                    @linkText
+                </a>
+                <br>
+                <span class="govuk-body-s govuk-!-margin-bottom-1 govuk-!-margin-top-1">
+                    @messages("chargeSummary.taxYear", chargeItem.taxYear.startYear.toString, chargeItem.taxYear.endYear.toString)
+                </span>
+            </td>
+            <td class="govuk-table__cell">@chargeItem.dueDate.getOrElse(chargeItem.documentDate).toLongDateShort</td>
+            <td class="govuk-table__cell govuk-table__cell--numeric">@chargeItem.originalAmount.abs.toCurrencyString</td>
```

Comment:
if we can cross check with BAs that we can use absolute value here? as I would expect this to be reflected in the requirements